native-gradle-plugin/src/samples/java-application-with-reflection/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'application'
+    id 'org.graalvm.buildtools.native'
+}
+
+repositories {
+    mavenCentral()
+}
+
+application {
+    mainClass.set('org.graalvm.demo.Application')
+}
+
+def junitVersion = providers.systemProperty('versions.junit')
+    .forUseAtConfigurationTime()
+    .orElse('5.7.2') // so that the sample runs if someone checks it out without the testing infra
+    .get()
+
+dependencies {
+    testImplementation(platform("org.junit:junit-bom:${junitVersion}"))
+    testImplementation('org.junit.jupiter:junit-jupiter')
+}
+
+test {
+    useJUnitPlatform()
+}
+
+nativeTest {
+    agent = true
+}
```

Comment:
Newline

Reply:
Any particular reason you want a newline?

Reply:
Mostly consistency, since we require newline for all of the java sources and all the other groovy `.gradle` files feature it. 
We picked up this standard from the oracle/graal repo, but I AFAIK it [originates from POSIX](https://stackoverflow.com/a/729795).

Reply:
fixed